debian/nnstreamer-nntrainer-trainer.install

```diff
@@ -0,0 +1 @@
+/usr/lib/nnstreamer/trainers/libnnstreamer_trainer_nntrainer.so
```

Comment:
So.. /usr/lib/nnstreamer/filters for the inference.. and /usr/lib/nnstreamer/trainers  for training?

Reply:
same, thank you.